### ISSUE TITLE
__init__.py Bug Report

### ISSUE THREAD

**Issue by BetaForever404** [ExcelentProgrammer/aiogram-template#12]
Based on the provided code, I can't see any syntax or runtime errors. However, there are a few potential issues that might cause problems depending on the context in which this code is used:

1. The `loader` module is assumed to contain a `dp` object that is an instance of `Dispatcher`. If the `loader` module has not been properly configured or the `dp` object has not been created, this code will raise an `AttributeError`.
2. Similarly, the `throttling` module is assumed to contain a `ThrottlingMiddleware` class. If the module has not been properly implemented or the class has not been defined, this code will raise a `ModuleNotFoundError` or `NameError`.
3. The `if __name__ == "__main__":` check is unnecessary and might cause a `NameError` if this module is being imported as a module rather than run directly. It's better to remove this check or replace it with a more appropriate check for the intended use case.

To summarize, the potential bugs in this code are:

* Improper configuration or implementation of the `loader` module
* Improper configuration or implementation of the `throttling` module
* Unnecessary or incorrect use of the `if __name__ == "__main__":` check.
